pkg/apis/core/validation/validation.go

```diff
@@ -3298,6 +3298,12 @@ func ValidatePodSecurityContext(securityContext *core.PodSecurityContext, spec *
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("runAsUser"), *(securityContext.RunAsUser), msg))
 			}
 		}
+		if securityContext.RunAsGroup != nil {
```

Comment:
if RunAsGroup is non-nil, do we require RunAsUser to be non-nil?

Reply:
no we dont